### PR TITLE
Rooms list

### PR DESCRIPTION
**Pull request by alex13bright**

### CODE REVIEW

File: app/components/room/header/Actions.tsx [hunk 1]
```diff
@@ -9,7 +8,7 @@ import { sidePaddingSize } from '~/components/room/header/headerStyles'
 const button = css`
   @media screen and ${widthAtLeast.lg} {
     padding: 10px;
-    width: 260px;
+    //width: 260px;
```
Comment (alex-adamant) [inline app/components/room/header/Actions.tsx:11]: So, in all the cases for commented out code is this something that you have to do temporary, but will definitely uncomment later? I wonder why is this the case though?

File: app/components/room/header/PagesNav.tsx [hunk 2]
```diff
@@ -88,14 +88,18 @@ const ActiveLink = styled.span`
 `
 
 const Item = styled.li``
-
+const lengthToColumns = (length: number): number => {
+  if (length === 3) return 3
```
Comment (alex-adamant) [inline app/components/room/header/PagesNav.tsx:92]: Interesting logic, but I guess this is what you needed here? If 3 -> 3, if even -> 2, in all other cases -> 1?

**Review by alex-adamant — APPROVED** [review]
In general looks good!
I see some commented out code, but I guess even if it's not needed it won't be too hard to remove later.
Same for some magic numbers. We have to many arbitrary numbers in code, but I guess it's fine for now.

File: app/components/room/header/Actions.tsx (hunk 1 above)
Comment (alex13bright) [reply app/components/room/header/Actions.tsx:11]: Это похоже из категории: "попробуем выключить, если ничего не сломается позже можно будет удалить" :)

File: app/components/room/header/PagesNav.tsx (hunk 2 above)
Comment (alex13bright) [reply app/components/room/header/PagesNav.tsx:92]: "3 -> 3, if even -> 2, in all other cases -> 1"
Да, логика именно такая. Не знаю можно ли это свернуть как то больше чем сейчас.

**Pull request merged**